### ISSUE TITLE
GET Endpoint to get student's data

### ISSUE THREAD

**Issue by rakaar** [kossiitkgp/kwoc-stats-api#9]
Background-
On the frontend side, we want to know whether the students have cleared mid evals or not, and similarly they have cleared end evals or not. Based on this info, appropriate message will be shown to them in their Dashboard like  "Thank you messsage for submitting end evals form" OR "you havent cleared mid evals message" 

We need a GET endpoint which tells frontend these 2 details of student - `mid_evals_cleared`, end _`evals_cleared`. 

Endpoint route can be `/is_cleared/student/<student_handle>` (Better naming is welcomed 😊)

The data should be read from Google Sheet

In short,
An endpoint with route - `/is_cleared/student/<student_handle>`
and uses the `student_handle` and looks up in the google sheet and returns the value of `is_mids_cleared`, `is_ends_cleared` columns

**Comment by Vipul-Bajaj** [comment]
I can pick this up, once PR #7 is merged
